docs/concepts/workloads/pods/disruptions.md

```diff
@@ -190,7 +190,7 @@ At this point, if an impatient cluster administrator tries to drain `node-2` or
 `node-3`, the drain command will block, because there are only 2 available
 pods for the deployment, and its PDB requires at least 2.  After some time
 
-asses, `pod-d` becomes available.
+passes, `pod-d` becomes available.
```

Comment:
not sure if this should be `assess` instead. @erictune ? also not sure why it is on a separate paragraph.